test/WebSites/ApplicationModelWebSite/Controllers/HomeController.cs

```diff
@@ -17,5 +20,22 @@ public object GetHelloWorld([FromHeader] string helloWorld)
         {
             return ControllerContext.ActionDescriptor.Properties["source"].ToString() + " - " + helloWorld;
         }
+
+        [HttpGet("Home/CannotBeRouted")]
+        [HttpGet("Home/CanBeRouted")]
+        [SuppressLinkGenerationConvention]
+        public object SuppressLinkGeneration()
```

Comment:
Should we add tests here too? https://github.com/aspnet/Mvc/tree/dev/test/WebSites/BasicWebSite/Controllers/LinkGeneration